judgels-frontends/raphael/public/var/conf/raphael.js.example

```diff
@@ -14,4 +14,12 @@ window.conf = {
     description:
       'Lorem ipsum dolor sit amet, consectetur adipisicing elit.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Lorem ipsum dolor sit amet, consectetur adipisicing elit.',
   },
+  termsAndConditions: `<p>By competing in TLX contests, you agree that:</p>
```

Comment:
/s/TLX/Judgels/g

Reply:
need further explanation

Reply:
Don't put "TLX" anywhere in the codebase